components/barcodes/barcode/overview.md

```diff
@@ -144,6 +75,62 @@ The Barcode methods are accessible through it's reference.
 }
 ````
 
+## Parameters
+
+The Blazor Barcode provides various parameters that allow you to configure the component:
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `RenderAs` | `enum` | Defines the preferred rendering mode - svg/canvas. |
```

Comment:
`enum` is not descriptive enough. Also, enums always have a default value that we should specify.

```suggestion
| `RenderAs` | `RenderingMode` enum <br /> `RenderingMode.Svg` | Defines the preferred rendering mode - svg/canvas. |
```